_targets.R

```diff
@@ -26,17 +80,14 @@ p2_targets_list <- list(
   tar_target(
     site_data_annotated,
     annotate_data(site_data_clean, site_filename = site_info_csv)
```

Comment:
You could probably combine all three of these steps `process_data()`, `annotate_data()` and `style_data()` into slightly larger `process_data()` function.